tests/test-grammar-integration.cpp

```diff
@@ -800,6 +948,165 @@ static void test_json_schema() {
         }
     );
 
+    test_schema(
+        "min -123 max 42",
```

Comment:
This schema looks identical to the one from the "simple min -123 max 42" test on line 249. Can they be merged together?

Reply:
Oops, reshuffled / merged them all, thanks!